WooCommerce/Classes/POS/Presentation/TotalsView.swift

```diff
@@ -230,6 +234,53 @@ private extension TotalsView {
     }
 }
 
+private extension TotalsView {
+    struct CardReaderViewLayout {
+        let backgroundColor: Color
+        let topPadding: CGFloat?
+        let bottomPadding: CGFloat?
+
+        static let primary = CardReaderViewLayout(
+            backgroundColor: .clear,
+            topPadding: nil,
+            bottomPadding: 8
+        )
+
+        static let dark = CardReaderViewLayout(
+            backgroundColor: Color(.quaternarySystemFill),
+            topPadding: 40,
+            bottomPadding: 40
```

Comment:
Consider renaming this – it makes it sound like it relates to dark/light mode to me

You could keep using `primary`, but replace `dark` with `emphasised`?

Other ideas:
- `defined`
- `delineated`
- `outlined`
- `structured`

No clear winner for me, but I prefer any of the above to `dark` to avoid confusion with dark/light mode.

Reply:
Okay, I get it renamed in a next PR.  I'm not a native speaker but `outlined` and `emphasised` sounds like good options.